### DISCUSSION TITLE
Find by method name

### DISCUSSION THREAD

**Discussion by gluonfield** [zed-industries/zed#6858] [Feature Requests]
Currently Zod has CMD+P search for file names. For productive work it would be good to have VSCode equivalent of searching by by method name (CMD+P # in VSCode).

<img width="603" alt="image" src="https://github.com/zed-industries/zed/assets/5672094/8e5338ec-c6dd-459e-816b-06a45a104b09">

**Comment by d1y** [comment]
Are you talking about `search symbols`?
Try use `cmd+t`

**Reply by gluonfield** [reply]
Thanks cmd+t works!
